test/wtr/check-tab.test.js

```diff
@@ -18,7 +18,7 @@ import sinon from 'sinon';
 import chromeMock from './mocks/chrome.js';
 import checkTab from '../../src/extension/check-tab.js';
 
-window.chrome = chromeMock;
+chrome = chromeMock;
```

Comment:
I think I would leave the `window.` here to make it explicit... feeld weird to just (re)define a global variable in a vacuum, wdyt?